metricsx/middleware.go

```diff
@@ -298,9 +341,9 @@ func (sw *Service) anonymizePath(path string, salt string) string {
 
 	for _, p := range paths {
 		p = strings.ToLower(p)
-		if len(path) == len(p) && path[:len(p)] == strings.ToLower(p) {
+		if path == p {
 			return p
-		} else if len(path) > len(p) && path[:len(p)+1] == strings.ToLower(p)+"/" {
+		} else if strings.HasPrefix(path, p) {
```

Comment:
I think the previous code was a bit complex here, but this should to exactly the same :sweat_smile: